tests/fullstack-test/expr/agg_bit_or_pushdown.test

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 PingCAP, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mysql> drop table if exists test.all_null
+mysql> create table test.all_null (a int, b int)
+mysql> alter table test.all_null set tiflash replica 1
+mysql> insert into test.all_null values(1, NULL), (1, NULL), (2, NULL), (2, NULL), (3, NULL);
+
+mysql> drop table if exists test.not_all_null
+mysql> create table test.not_all_null (a int, b int)
```

Comment:
Can the code support decimal type now？

Reply:
Yes, and string types are also supported. It depends on whether `CAST` function can support the conversion or not.